### PR TITLE
fix(ui-ux): remove border on ThemedTouchableOpacityV2 from marketplace screen

### PR DESCRIPTION
**Pull request by G9000**
#### What kind of PR is this?:
/kind fix

#### What this PR does / why we need it:
Remove border on ThemedTouchableOpacityV2 from marketplace screen

#### Which issue(s) does this PR fixes?:
Fixes #

#### Developer Checklist:
- [x] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx [hunk 1]
```diff
@@ -112,33 +111,39 @@ export function MarketplaceScreen (): JSX.Element {
...
 
-function ExchangeItemRow ({ image, name, url, testID, border }: ExchangeProps & { testID: string } & { border: boolean}): JSX.Element {
+function ExchangeItemRow ({ image, name, url, testID, isLast }: ExchangeProps & { testID: string } & { isLast: boolean}): JSX.Element {
     return (
-      <ThemedTouchableOpacityV2
-        onPress={async () => await openURL(url)}
-        style={tailwind('flex flex-row mx-5 py-4 items-center justify-between', { 'border-b-0.5': border })}
-        testID={testID}
+      <ThemedViewV2
+        style={tailwind('border-b-0.5', { 'border-0': isLast })}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx:117]: `ThemedViewV2` has no border by default
```suggestion
        style={tailwind({ 'border-b-0.5': !isLast })}
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx [hunk 2]
```diff
@@ -112,33 +111,39 @@ export function MarketplaceScreen (): JSX.Element {
...
+          style={tailwind('flex flex-row items-center justify-between py-4 border-0')}
+          testID={testID}
+        >
+          <View style={tailwind('flex flex-row items-center')}>
+            <Image
+              source={image}
+              style={tailwind('h-6 w-6')}
+            />
+            <ThemedTextV2
+              dark={tailwind('text-mono-dark-v2-900')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx:132]: redundant styling for light + dark mode

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx (hunk 1 above)
Comment (G9000) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/MarketplaceScreen.tsx]: updated

**Comment by kyleleow** [comment]
Let's close this and PR #3171 , and work on creating a reusable component that has this behaviour

**Pull request closed without merging**